Gemfile

```diff
@@ -10,6 +10,8 @@ def lock_manager_location_for(place)
   end
 end
 
+gem "aws-sdk", "~> 2"
```

Comment:
Should this be constrained a little more, to `2.3` or `2.3.30`? I'm not sure whether the AWS SDK is semver or what their track record for API breakage is, but for Vanagon we try to err on the side of caution.